### PR TITLE
[US-684] Example for Report From CSV

### PR DESCRIPTION
**Pull request by kellemNegasi**

### CODE REVIEW

File: report/README.md [hunk 1]
```diff
@@ -7,3 +7,4 @@ The example explains how to create PDF reports using UniPDF, it also shows you h
 - [pdf_report.go](pdf_report.go) The example showcases PDF report generation with UniPDF's creator package. The output is saved as unidoc-report.pdf which illustrates some of the features of the creator.
 - [pdf_tables.go](pdf_tables.go) The example showcases PDF tables features using UniPDF's creator package. The output is saved as UniPDF-tables.pdf which illustrates some of the features of the creator.
 - [pdf_custom_toc.go](pdf_custom_toc.go) The example showcases the capabilities of generating PDF custom table of contents layout. The output is saved as pdf-custom-toc.pdf.
+- [pdf_report_from_csv.go](pdf_report_from_csv.go) This example showcases how to prepared report from csv data.
```
Comment (anovik) [inline report/README.md:10]: probably misprint, should be "how to prepare"

File: report/pdf_report_from_csv.go [hunk 2]
```diff
@@ -0,0 +1,396 @@
+package main
```
Comment (anovik) [inline report/pdf_report_from_csv.go:1]: Do we need some description or copyright in the beginning?

File: report/pdf_report_from_csv.go [hunk 3]
```diff
@@ -0,0 +1,396 @@
...
+	DateOfSale      time.Time
+	QuantitySold    float64
+	SalePrice       float64
+	SalespersonName string
+}
+
+func main() {
+	robotoFontRegular, err := model.NewPdfFontFromTTFFile("./Roboto-Regular.ttf")
+	if err != nil {
+		panic(err)
```
Comment (anovik) [inline report/pdf_report_from_csv.go:49]: Can we have here and below error logging instead of panic?

**Review by anovik — COMMENTED** [review]
Mostly looks good, could you please address several comments?

**Review by anovik — APPROVED** [review]

**Review by sampila — APPROVED** [review]
looks good to me

**Pull request merged**